docusign/docusign.lua

```diff
@@ -0,0 +1,79 @@
+--[[
+
+# [DocuSign API DOCUMENTATION](https://docs.docusign.com/esign/guide/usage/quickstart.html) connector.
+
+### Authorization
+
+Authorization use [Basic authentication](https://admindemo.docusign.com).
+
+## Input parameters
+
+* `username` - application user email used to authorize the call.
+* `password` - user password used to authorize the call
+* `integratorKey` - integration and authenticate with the DocuSign platform
+
+* api_endpoint - API endpoint URI, e.g. `user`.
+* method - optional request method, defauls to `GET`, could be `GET`, `POST`, `PUT`, `DELETE`.
+* post_data - optional JSON string used to post data, e.g. `{"blog":{"title":"Test Title"}}` structure used to create a Blog item.
```

Comment:
refer to DocuSign example https://docs.docusign.com/assets/quickstart/send_signing_request_by_email.json